src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/RunnableProjectGenerator.cs

```diff
@@ -256,6 +256,77 @@ public bool TryGetTemplateFromConfigInfo(IFileSystemInfo templateFileConfig, out
                 };
                 SimpleConfigModel templateModel = SimpleConfigModel.FromJObject(templateFile.MountPoint.EnvironmentSettings, srcObject, configModifiers, localeSourceObject);
 
+                //Do some basic checks...
+                List<string> errorMessages = new List<string>();
+                List<string> warningMessages = new List<string>();
+                if (string.IsNullOrEmpty(templateModel.Identity))
+                {
+                    errorMessages.Add(string.Format(LocalizableStrings.Authoring_MissingValue, "identity", templateFile.FullPath));
+                }
+
+                if (string.IsNullOrEmpty(templateModel.Name))
+                {
+                    errorMessages.Add(string.Format(LocalizableStrings.Authoring_MissingValue, "name", templateFile.FullPath));
+                }
+
+                if ((templateModel.ShortNameList?.Count ?? 0) == 0)
+                {
+                    errorMessages.Add(string.Format(LocalizableStrings.Authoring_MissingValue, "shortName", templateFile.FullPath));
```

Comment:
Should this be just a warning? I wasn't clear if we'd finalized whether short name should be required or not.